XamarinCommunityToolkitSample/Resx/AppResources.Designer.cs

```diff
@@ -10,48 +10,35 @@
 
```

Comment:
Wow, this file is regenerated again...

@jfversluis we should reach an agreement about this file structure. That's now very good when this file is regenerated each time a new PR is coming :)